app/black_spots/models.py

```diff
@@ -0,0 +1,38 @@
+import uuid as uuidlib
+
+from django.conf import settings
+from django.contrib.gis.db import models
+
+from ashlar.models import AshlarModel
+
+
+class BlackSpot(AshlarModel):
+    """A black spot -- an area where there is an historical/statistical concentration of records"""
+
+    #: Buffered road segment polygon where black spot analysis is performed
+    geom = models.PolygonField(srid=settings.ASHLAR['SRID'])
+
+    #: Number that determines the severity of this black spot. May be used for color-coding on map.
+    severity_score = models.FloatField()
+
+    #: Number of records accounted for in the polygon while analyzing
+    num_records = models.PositiveIntegerField()
```

Comment:
Is the reason you made these fields rather than `@property` because it would be easier to display black spots using windshaft? That makes sense to me but I initially thought it might be better to do them as `@property`, before I thought about the Windshaft case, so I'm curious what your reasoning was.